<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test results</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="floatable">
      <bool>false</bool>
     </property>
     <addaction name="actionSaveAll"/>
     <addaction name="actionSaveSelected"/>
     <addaction name="actionOpenTracker"/>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTreeWidget" name="resultsTree">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QTextBrowser" name="resultText">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSaveAll">
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Save All Results</string>
   </property>
  </action>
  <action name="actionSaveSelected">
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Save Selected Results</string>
   </property>
  </action>
  <action name="actionOpenTracker">
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Open Issue Page</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
